### PR TITLE
Made css stylings more consistent

### PR DESCRIPTION
**Pull request by Mingling94**
I thought it odd that rows in the plugins were larger than the 60px height that were uniform to our buttons and headers so I made them all 60px.

### CODE REVIEW

**Comment by DavidVorick** [comment]
did this change the look at all? Can you post screenshots of before/after?

At some point, it would be nice to have some sort of framework that produced screenshots of the changes on various systems and resolutions. heh, more testing. (this is not a near-future need)

**Comment by Mingling94** [comment]
It's minute, but rows are even in this branch and plugin buttons highlight to a uniform color:
sia_010
sia_022

As opposed to in master where the bars are misaligned compared to the header/sidebar buttons and the hover shading is defined per plugin and is slighlty lighter, as demonstrated by the `Add Folder` button.
sia_019
sia_015

**Comment by DavidVorick** [comment]
I'm not sure that you want this stuff lined up. They are separate parts of the UI, and when they don't line up it's much more clear that there's a difference between them. Would be nice to get an opinion from a designer.

**Comment by seveibar** [comment]
I could go wrong either way on this one, David makes a good point about different functional parts of the UI. But I like the appearance of the symmetry.

I think the symmetry might be better with a smallish border to distinguish the active area.

If on the fence, its probably a good idea to trust the in-house designer (Ming here) :)

**Comment by DavidVorick** [comment]
@pmknutsen since you designed it, you're actually the in-house designer here

**Comment by pmknutsen** [comment]
Some general comments (from a non-designer, but someone knowledgeable on visual processing):

Aligning elements has a visually calming effect. This is especially true when elements are perceived as belonging to the same 'group'. Our visual system groups by all kinds of features, including spatial features such as nearby edges with matching orientations. In this case, you have multiple horizontal lines that almost, but not quite, merge. So, we tend to perceive something continuous, because the elements almost line up. This triggers our attention to a perceived misalignment. In fact, the exact elements Ming aligned have also irked me on a few occasions. In a UI, you generally don't want a user's attention drawn to anything that is not a 'feature', i.e. you want a focus on application usability and not on design (i.e. a good design is something that does not draw any kind of attention away from the usability aspect).

Following this, a designer can also 'over-align'. This happens when you align elements that otherwise would not be visually grouped, e.g. because they are visually too distinct (e.g. don't share color, shape, orientation, too far apart etc).

I don't think it is important here that the menu and body of the UI are different 'parts' of the UI, and thus should not be aligned. You should align not because of function, but because you don't want to create a visual distraction if you otherwise did not align.

Since at least two people (Ming and I) were both irked by this small mismatch, I think that tells us that these elements should be aligned.

So, long story short, I too vote for alignment.

**Comment by Mingling94** [comment]
Wow thanks for the insight @pmknutsen!

**Comment by DavidVorick** [comment]
From Jing:

```
okay so
two design decisions i disagree with:
regardless of whether they line up or not, the bottom lines of the objects on the left should not extend all the way aross the page
across*
a little bit of lining up is ok
let me screenshot you an example

http://imgur.com/Q5pJwWC

search bar is aligned, but that's ok
site objects are not aligned
does that make sense?
```

image:
12214206_10153369626023795_568879534_o

**Comment by DavidVorick** [comment]
Sorry, had a few meetings, didn't get a chance to read @pmknutsen's comment until now.

Jing's suggestions solve this problem by preventing the lines from going to the edge, which means your eyes don't try to line things up anymore

**Comment by Mingling94** [comment]
So I think this is what Jing meant, specifically with the about page.
sia_026

**Comment by DavidVorick** [comment]
I'm just nitpicking at this point, but since it's a style PR I guess that makes sense. I don't think you want the underline to line up perfectly with the text, if you look at Jing's screenshot, the line is actually larger than the text by a little bit.

**Comment by Mingling94** [comment]
Yo, out of left-field here, but what about
sia_028

Otherwise this pr is updated with those slightly longer lines you wanted.
sia_029

**Comment by DavidVorick** [comment]
Gotta say, now that I'm seeing it, I think it looks a lot better without the lines extended. I guess I'm not a designer lol.

I do like the idea of adding a text blurb with some links, but the way you did it above doesn't look good at all. Also, I'm not sure I like the version numbers in a capsule, seems like the wrong place to put them.

**Comment by Mingling94** [comment]
Yeah that first screenshot is just for concept. Otherwise this PR looks like the second screenshot in my previous comment, can it be merged?
